tests/functional/configured_endpoint_urls/profile-tests.json

```diff
@@ -73,6 +73,13 @@
         "default": {},
         "endpoint_url_provided":{
           "endpoint_url": "https://client-config.endpoint.aws"
+        },
+        "ignore_configured_endpoint_urls": {
+          "ignore_configured_endpoint_urls": "true"
```

Comment:
Should these booleans instead of the string "true"?

Reply:
I used string "true" previously, but I suppose that's because it was for contents of the config file or the environment variable value. Boolean `true` should work here.

Example:

https://github.com/boto/botocore/blob/132f004f6d84ccee1aef91990a0a03a99dfb1927/tests/functional/configured_endpoint_urls/profile-tests.json#L46

Reply:
I think more of my point is that the expected `Config` interface is to use a boolean and not the string (e.g. `Config(ignore_configured_endpoint_urls=True`) and so it makes more sense to have the value as the expected type here to avoid having to do a type conversion of the string `"true"`. I also suspect other projects using this file would want to avoid doing the conversion as well.